product_docs/docs/edbcloud/beta/getting_started/01a_managing_quotas.mdx

```diff
@@ -0,0 +1,187 @@
+ResourceType     Locations    Name             Zones    Restrictions
+---------------  -----------  ---------------  -------  -----------------------------------------------------------------------
+virtualMachines  eastus2      Standard_DS2_v2  1,2,3    NotAvailableForSubscription, type: Zone, locations: eastus2, zones: 3,2
+```
+The presence of entries in Restriction column would block EDB Cloud from deploying Postgres cluster in the specified region. The value in in Restrictions column must be `None` as in the example below:
+
+```
+$ az vm list-skus -l westus2 --zone --size Standard_DS2_v2
+ResourceType     Locations    Name             Zones    Restrictions
+---------------  -----------  ---------------  -------  --------------
+virtualMachines  westus2      Standard_DS2_v2  1,2,3    None
+```
+
+
+
+#### Checking Azure Service Providers registrations using Azure CLI
+
+The presence of entries in Restriction column would block EDB Cloud from deploying Postgres cluster in the specified region. The value in in Restrictions column must be `None` as in this example:
+
+```
+$ az vm list-skus -l westus2 --zone --size Standard_DS2_v2
+
+ResourceType     Locations    Name             Zones    Restrictions
+---------------  -----------  ---------------  -------  --------------
+virtualMachines  westus2      Standard_DS2_v2  1,2,3    None
+```
+
+## Remediate quota issues
+
+### Increasing Public IP Address quota
+
+You can increase the number of public IP addresses for your account either by using Azure's portal if you have appropriate privileges or by submitting a support request. See:
+
+- [Request Networking quota increase at subscription level using Help + support](https://docs.microsoft.com/en-us/azure/azure-portal/supportability/networking-quota-requests#request-networking-quota-increase-at-subscription-level-using-help--support)
+
+- [Request networking quota increase at subscription level using Usages + quotas](https://docs.microsoft.com/en-us/azure/azure-portal/supportability/networking-quota-requests#request-networking-quota-increase-at-subscription-level-using-usages--quotas)
+
+
+### Increasing vCPU quota
+
+You can increase the number of ESv3 Series virtual machines per region for your account either by using Azure's portal if you have appropriate privileges or by submitting a support request. See:
+
+- [Request a quota increase by region from Help + support](https://docs.microsoft.com/en-us/azure/azure-portal/supportability/regional-quota-requests#request-a-quota-increase-by-region-from-help--support)
+
+- [Request a quota increase by region from subscriptions](https://docs.microsoft.com/en-us/azure/azure-portal/supportability/regional-quota-requests#request-a-quota-increase-by-region-from-subscriptions)
+
+### Fixing issues with SKU restrictions
+
+[Open a support request](https://docs.microsoft.com/en-us/troubleshoot/azure/general/region-access-request-process)  to remove SKU restrictions in the designated region.
```

Comment:
Does this actually work? My understanding is that Azure will just tell you to wait until they're available.

Reply:
I don't understand the problem well enough to say. Need @xugy99, @nstott, or others to chime in.

Reply:
>did we test this? 

No, we never test that. The only once we encountered that, the support suggested the customer to alternate to another location(region). 

We give this recommendation of "open a support request" all because of it is what the Azure documentation is suggesting, reading [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/troubleshooting/error-sku-not-available#solution-3---azure-portal)

> This article describes how to resolve the SkuNotAvailable error. If you're unable to find a suitable SKU in that 
> region/zone or an alternative region/zone that meets your business needs, submit a [SKU request](https://docs.microsoft.com/en-us/troubleshoot/azure/general/region-access-request-process) to Azure Support.

Unfortunately, the link of [SKU request](https://docs.microsoft.com/en-us/troubleshoot/azure/general/region-access-request-process) points to some quite generic page.

The interesting behavior is: the SKU availability from Azure looks like more differs from customer to customer: we observed when one VM SKU (e.g. EsV3) is not available in `eastus` to Marc's account, while it is available to our Azure account.


>what is the alternative? If it does not work and Azure just tells you to wait. Should we document this as something that the user can fix or mention it as a limitation. 

As long as we declare we can support alternative instance-type rather than only EsV3 for the PG cluster type, beside pushing to Azure, I don't see there is anything customer can do, after I read the doc of https://docs.microsoft.com/en-us/azure/azure-resource-manager/troubleshooting/error-sku-not-available#solution-3---azure-portal

Reply:
@ba-edb Do you still want to recommend the user to raise a support request, considering that might be our only alternative?